src/coreclr/src/vm/typedesc.cpp

```diff
@@ -74,10 +57,10 @@ TypeHandle TypeDesc::GetBaseTypeParam()
 
```

Comment:
This is same as `GetMethodTableOfRootElementType`. It would be nice if it is called same (`Root` or `Base`) in both cases.

Reply:
I like `Root` more. - A better hint that it is recursive (vs. just one layer). Will rename this one.